aliases.yaml

```diff
@@ -77,8 +77,18 @@ cmds:
     cmd: describe
   - short: g
     cmd: get
+  - short: g
+    cmd: get
+    suffix:
+      short: g
+      cmd: " -o wide | grep"
   - short: ga
     cmd: "get --all-namespaces -o wide"
+  - short: ga
```

Comment:
TODO: remove this duplicate 2 lines

Reply:
In retrospect. This might require some refactoring to make it all work. I'm not sure

Reply:
Yea, I think so. They end up with two different commands though. I think we would have to have a more complicated design for the yaml to make it work. Im not opposed to that. But this seems a little more explicit.